src/engraving/accessibility/accessibleroot.cpp

```diff
@@ -170,3 +171,64 @@ void AccessibleRoot::setMapToScreenFunc(const AccessibleMapToScreenFunc& func)
 {
     m_accessibleMapToScreenFunc = func;
 }
+
+bool AccessibleRoot::isRangeSelection() const
+{
+    if (auto focusedItem = m_focusedElement.lock()) {
+        const EngravingItem* item = focusedItem->element();
+        if (item) {
+            return item->score()->selection().isRange();
+        }
+    }
+    return false;
+}
+
+QString AccessibleRoot::rangeSelectionInfo()
+{
+    auto focusedItem = m_focusedElement.lock();
+    if (!focusedItem) {
+        return QString();
+    }
+    const EngravingItem* item = focusedItem->element();
+    if (!item) {
+        return QString();
+    }
```

Comment:
Instead of using `m_focusedElement` I think this is sufficient:
```
Score* score = element()->score();

// use later for score->selection() and score->staff()
```